### ISSUE TITLE
Section for What forces layout / reflow

### ISSUE THREAD

**Issue by yeomann** [GoogleChrome/css-triggers#23]
Hi guys, from this question[ stack overflow question](http://stackoverflow.com/questions/37497307/layout-being-triggered-on-css-transform/37497554#37497554), may i suggest if we can add section for something like **Layout trashing**. For example, On site it is mentioned that transform property doesn't cause geometry change i.e no layout re-flow but i think it's worth mentioning that if its called through JavaScript then it does! 
Let me know you thoughts if i can contribute as well or if i am wrong?

**Comment by oldergod** [comment]
Do you have sample code that would prove the fact that "transform causes
layout to ruin" ?

**Comment by surma** [comment]
Changing a transform (even through JS) does not force relayout or repaint in Chrome. The gist by Paul Irish says that `getComputedStyle()` does cause relayout *if* a transform is set. 

Currently, csstriggers shows that Gecko, Edge and Webkit do relayout when you change `transform`. However, @paullewis and I are believe that to be a bug of the respective DevTools and is not actually true.

**Comment by gregwhitworth** [comment]
Hey @surma and @paullewis I do think it's worth adding this section. We should document what APIs cause a reflow, eg: getBoundingClientRect() offsetHeight, etc as we see the common read-modify-write pattern on the web and it would be nice if this was included.

**Comment by surma** [comment]
For reference: [@paulirish’s collection of JS APIs that cause layout](https://gist.github.com/paulirish/5d52fb081b3570c81e3a)

**Comment by gregwhitworth** [comment]
Ok, I'm not a hardcore designer but I know my away Photoshop - here's a quick mockup, don't focus on the data it's merely for finalizing the design. Let me know your thoughts. I recommend switching the domain to *triggers.com which most are actually available since technically these aren't CSS triggers but do force a style recalc and possibly layout. Let me know your thoughts @surma and @paullewis 

![css-triggers](https://cloud.githubusercontent.com/assets/865244/22177407/8b644414-dfd1-11e6-97d1-35ba7f70d894.png)

**Comment by surma** [comment]
Yeah, @paullewis had already been messing around with potential domain names. Any suggestions welcome. EngineTriggers doesn’t sound too bad, but I’d like to convey that it’s web-related somehow.

Design looks A+ to me, great stuff :)

**Comment by gregwhitworth** [comment]
This feels like the need for a twitter poll, UA Triggers, Browser Triggers, Pipeline Triggers

**Comment by surma** [comment]
Ha, let’s [see what happens](https://twitter.com/DasSurma/status/822901883408908292).

**Comment by surma** [comment]
So, technically “Browser triggers” won, but I liked the idea of just getting “jstriggers.com” (which I just bought) and having both URLs link to the same site. WDYT?

**Comment by oldergod** [comment]
Browser triggers is a mouthful to me. I like the idea having multiple domains linking to a same site.

**Comment by gregwhitworth** [comment]
Be honest with everyone, trigger just makes me think of [chiggers](https://en.wikipedia.org/wiki/Trombiculidae) so whatever works for everyone else I'm good with. Just have the title swap out then when you change the tab * Triggers. I'm assuming you all are able to insert the data from Paul's post into the format?
